### PR TITLE
Test tmdb

### PR DESCRIPTION
**Pull request by unruffled-nightingale**
Integrated and tested tmdb in gather.py

### CODE REVIEW

File: py/test_gather.py [hunk 1]
```diff
@@ -53,6 +61,7 @@ def test_get_movie_data_omdb(self):
         omdb = self.get.apis[0]
         api_param = ('tt4285496',)
         data = self.get.get_movie_data(omdb, api_param)
+        print(data)
```
Comment (teddelin) [inline py/test_gather.py]: I would say no prints in master that are not part of the process

File: py/get_tmdb.py [hunk 2]
```diff
@@ -29,13 +35,17 @@ def split_movie_data(self, imdb_id, api_data):
...
         videos = api_data["videos"]["results"]
-        keywords = api_data["kewords"]["keywords"]
+        keywords = api_data["keywords"]["keywords"]
         lists = api_data["lists"]["results"]
         changes = api_data["changes"]["changes"]
         trailers = api_data["trailers"]["youtube"]
-        release_dates_raw = api_data["release_dates"]["results"]["release_dates"]
+        release_dates = api_data["release_dates"]["results"][0]["release_dates"]
+
+        for dict in cast_data:
```
Comment (teddelin) [inline py/get_tmdb.py]: I don't know if using dict like this does something special but using default keywords for variable names are usually not preferable.

**Review by teddelin — APPROVED** [review]

**Review by teddelin — COMMENTED** [review]
Just have a look at my comments

File: py/get_tmdb.py (hunk 2 above)
Comment (unruffled-nightingale) [reply py/get_tmdb.py]: i agree. what index name would you use to describe a dictionary ?

File: py/test_gather.py (hunk 1 above)
Comment (unruffled-nightingale) [reply py/test_gather.py]: Agreed, will remove.

File: py/get_tmdb.py (hunk 2 above)
Comment (teddelin) [reply py/get_tmdb.py]: You would use a name that describes the data such as cast_entry or something, not sure what's in the variable.

**Pull request merged**